### PR TITLE
created home footer

### PR DESCRIPTION
**Pull request by gissellediaz**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (https://trello.com/c/tntRNB4t/7-home-footer)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/components/footer/Footer.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          <i className="fas fa-search"/>
+        </div>
+        <div className="Icon">
+          <i className="far fa-plus-square"/>
+        </div>
+        <div className="Icon">
+          <i className="far fa-heart"/>
+        </div>
+        <div className="Icon">
+          <i className="fas fa-user"/>
```
Comment (davidfigueroar9) [inline src/components/footer/Footer.jsx]: can you change `fas` for `far`?, then the user icon don't will be filled.

File: src/components/footer/Footer.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        </div>
+      </div>
+    </div>
+  );
+}
+  
+  export default Footer;
+
```
Comment (davidfigueroar9) [inline src/components/footer/Footer.jsx]: so many lines after `export default`. we should have only one new line at the end of file

Comment (gissellediaz) [reply src/components/footer/Footer.jsx]: done

Comment (gissellediaz) [reply src/components/footer/Footer.jsx]: done

File: src/components/footer/Footer.jsx (hunk 1 above)
Comment (gissellediaz) [reply src/components/footer/Footer.jsx]: done!

**Pull request merged**